### ISSUE TITLE
Doesn't works with React 18

### ISSUE THREAD

**Issue by aaron5670** [prismicio/prismic-next#16]
### Versions

- package_name: 0.1.0
- node: 16.14.2

### Reproduction
When running ``npm install``

### Steps to reproduce
Run npm install with Next.js 12.1.2 and @prismicio/next 0.1.0

### What is actually happening?

```
C:\Users\aaron.vandenberg\WebstormProjects\project-v2-react18>npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: website@undefined
npm ERR! Found: react@18.0.0
npm ERR! node_modules/react
npm ERR!   react@"^18.0.0" from website@undefined
npm ERR!   apps/website
npm ERR!     website@undefined
npm ERR!     node_modules/website
npm ERR!       workspace apps\website from the root project  
npm ERR!   peer react@"^17.0.2 || ^18.0.0-0" from next@12.1.0
npm ERR!   node_modules/next
npm ERR!     next@"12.1.0" from website@undefined
npm ERR!     apps/website
npm ERR!       website@undefined
npm ERR!       node_modules/website
npm ERR!   @prismicio/next@"0.1.0" from website@undefined
npm ERR!   apps/website
npm ERR!     website@undefined
npm ERR!     node_modules/website
npm ERR!       workspace apps\website from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
```

**Comment by angeloashmore** [comment]
Hey @aaron5670, thanks for the report!

Once `@prismicio/react` is updated to support React 18 (already in `master`, will be published shortly), `@prismicio/next` will be updated to support React 18.

If you need an immediate workaround, you can run use `--legacy-peer-deps` when installing packages:

```sh
npm install --legacy-peer-deps
```

I recommend you wait for the official update, but you can "force" the upgrade with the above command. `@prismicio/next` *should* already be compatible with React 18, but some testing will be done before the updated version is published.

Thanks again!

**Comment by angeloashmore** [comment]
React 18 support is available in `v0.1.1`. 🎉

See #17 for more details.

Could you try it out and let me know if it works? I'm going to close this issue, but let me know if you see and issue and I will reopen it.
